projects/storefrontapp-e2e-cypress/cypress/helpers/asm.ts

```diff
@@ -93,24 +110,15 @@ export function assertCustomerIsSignedIn() {
   cy.get('cx-login div.cx-login-greet').should('exist');
 }
 
-export function deleteFirstAddress() {
-  interceptDelete('deleteAddresses', '/users/*/addresses/*?lang=en&curr=USD');
-  interceptGet('fetchAddresses', '/users/*/addresses/*?lang=en&curr=USD');
-
-  const firstCard = cy.get('cx-card').first();
-  firstCard.contains('Delete').click();
-  cy.get('.cx-card-delete button.btn-primary').click();
-  cy.wait('@deleteAddress');
-  cy.wait('@fetchAddresses');
-}
-
 export function testCustomerEmulation() {
+  let customer: SampleUser;
+
   it('should test customer emulation', () => {
-    let customer = getSampleUser();
-    checkout.registerUser(false, customer);
+    checkout.visitHomePage();
+
+    customer = checkout.registerUser(false);
```

Comment:
Not within the scope go this PR, but I guess we should move checkout.visitHomePage() in navigation.ts at some point in the future.